variable.go

```diff
@@ -61,33 +62,29 @@ type Variable struct {
 	Sensitive bool         `jsonapi:"attr,sensitive"`
 
 	// Relations
-	Workspace *Workspace `jsonapi:"relation,workspace"`
+	Workspace *Workspace `jsonapi:"relation,configurable"`
```

Comment:
Interesting, are you sure this works as expected?

Reply:
Seems you didn't answer this question yet. Can you share the reasoning for changing this as I noticed you used `configurable` in the new `PolicySetParams` struct as well?

Reply:
The relation name was changed in the API since it can be either workspaces or policy sets.

Reply:
Check... I'm a bit surprised that it works as expected in that case, but cool if it does 👍

Reply:
Sorry, I thought I had replied! The models behind the scene utilize a polymorphic relationship, which is based on a "configurable" type that can either be a workspace or policy set, as @thrashr888 said.

Reply:
Yeah, no problem. But I just wonder if this works as expected as I think to remember that the JSONAPI package doesn't work with polymorphic types (as it's all in Go right). But if it does and all the tests pass, then I guess I'm wrong and this is fine.

Reply:
I think it's most likely because of the way that the API is interacted with. The `configurable` isn't sent in the create/update requests and is actually handled within the API according to the route.

Reply:
I just added a couple of assertions to ensure that the policy set id is set and retrieved as expected.